flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/source/ValuesDataSource.java

```diff
@@ -238,11 +258,23 @@ private static class EventIteratorReader
         // position for this Split to fail
         private final int failAtPos;
 
+        private final ValuesDataSourceHelper.SourceEventType eventType;
```

Comment:
Event is our core concept. The name `SourceEventType` makes it easy for people to think that this source only produce `SchemaChangeEvent` or `DataChangeEvent`. In fact, the word should describe the input data set of ValueSource. Should we consider simply using `EventSetId`?